musicbrainz/csv/convert_to_csv.py

```diff
@@ -0,0 +1,101 @@
+# This file converts a JSON Lines file from MusicBrainz to a CSV file
+# For OpenRefine Reconciling
+# Input file can be downloaded in
+# https://data.metabrainz.org/pub/musicbrainz/data/json-dumps/20240522-001002/
+# Output file can be imported to OpenRefine
+import json
+import copy
+
+header = ["id"]
+values = []
+
+with open("test2", "r") as f:
+    json_data = [json.loads(m) for m in f]
+
+# print(json_data)
+
+
+def extract(data, first_level: bool = True, key: str = "", value: dict = {}):
+    if key != "":
+        first_level = False
+    if "relations" in key or "aliases" in key:
```

Comment:
`key` is a string, so you are seeing if these are substrings?

Reply:
Yes.

Reply:
Since relations are external links to websites and aliases have a bunch of repeated names, I chose to ignore them when testing.

Reply:
So you want this to return `True` if `key="here_is_some_alias"?

Reply:
Yes.